### PR TITLE
Contribution logoupdate

### PR DESCRIPTION
**Pull request by JipeiWang**
### Motivation
Invite Pulsar users to update their company logos on Pulsar website.

### Modifications
Add workflow of updating logos.

### CODE REVIEW

File: site2/website/contributing.md [hunk 1]
```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
 
 Follow the [README](https://github.com/apache/pulsar/tree/master/site2) for making contributions to the website.
 
+#### Updating user logo
+The Apache Pulsar Community welcomes all users to update their company logos on Pulsar website. If your team or organization has adopted Pulsar in production, please update your company logos on [Pulsar website](https://pulsar.apache.org/en/powered-by/).
+
+It only takes 4 steps to update your company logo:
+1. Open [link]: (https://github.com/apache/pulsar/blob/master/site2/website/data/users.js) in your browser, click “edit” to fork Pulsar repo, create a new branch and edit this file.
```
Comment (sijie) [inline site2/website/contributing.md:302]:
```suggestion
1. Open [link](https://github.com/apache/pulsar/blob/master/site2/website/data/users.js) in your browser, click “edit” to fork Pulsar repo, create a new branch and edit this file.
```

**Review by sijie — APPROVED** [review]
@JipeiWang Thank you for putting this guideline! That's useful! I only have one small comment. Otherwise, it looks good.

File: site2/website/contributing.md [hunk 2]
```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
 
 Follow the [README](https://github.com/apache/pulsar/tree/master/site2) for making contributions to the website.
 
+#### Updating user logo
+The Apache Pulsar Community welcomes all users to update their company logos on Pulsar website. If your team or organization has adopted Pulsar in production, please update your company logos on [Pulsar website](https://pulsar.apache.org/en/powered-by/).
```
Comment (Jennifer88huang) [inline site2/website/contributing.md:299]:
```suggestion
The Apache Pulsar Community welcomes all users to update their company logos on Pulsar website. If your team or organization has adopted Pulsar in production, you can update your company logos on [Pulsar website](https://pulsar.apache.org/en/powered-by/).
```

File: site2/website/contributing.md [hunk 3]
```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
...
+
+It only takes 4 steps to update your company logo:
```
Comment (Huanli-Meng) [inline site2/website/contributing.md:301]:
```suggestion
To update your company logo, follow these steps:
```

File: site2/website/contributing.md [hunk 4]
```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
...
+    {
+        name: 'StreamNative',
+        url: 'https://streamnative.io/',
+        logo: 'https://streamnative.io/static/images/streamnative-logo-white-bakground.png',
+    }
+`
+If your company logo is in white color, add another line after the logo link in JSON file:
+`logo_white: true`
+
+3. Commit the changes in your account’s repo, publish the branch and create pull request.
```
Comment (Huanli-Meng) [inline site2/website/contributing.md:316]:
```suggestion
3. Commit your updates, create a PR, and then publish the PR to the master repository.
```

File: site2/website/contributing.md [hunk 5]
```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
...
+4. Wait for review. If the PR works, your pull request will be approved and merged by reviewers.
```
Comment (Huanli-Meng) [inline site2/website/contributing.md:317]:
```suggestion
4. Update comments, if any. If no more comment, ask reviewers to approve the PR and merge the PR to the master.
```

File: site2/website/contributing.md [hunk 6]
```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
...
+It only takes 4 steps to update your company logo:
+1. Open [link](https://github.com/apache/pulsar/blob/master/site2/website/data/users.js) in your browser, click “edit” to fork Pulsar repo, create a new branch and edit this file.
+2. Add your company’s information at the end of file. 
+
+Example:
+`
+    {
+        name: 'StreamNative',
+        url: 'https://streamnative.io/',
+        logo: 'https://streamnative.io/static/images/streamnative-logo-white-bakground.png',
```
Comment (Huanli-Meng) [inline site2/website/contributing.md:310]:
```suggestion
        logo: 'https://streamnative.io/static/images/streamnative-logo-white-bakground.png',
        logo_white: true
```

File: site2/website/contributing.md [hunk 7]
```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
...
+    }
+`
+If your company logo is in white color, add another line after the logo link in JSON file:
```
Comment (Huanli-Meng) [inline site2/website/contributing.md:313]: > **Note**  
The `logo_white` parameter is set to `true` only when your company logo uses a white background.

File: site2/website/contributing.md [hunk 8]
```diff
@@ -295,6 +295,27 @@ the static content is generated by CI job and merged into the `asf-site` branch.
 
 Follow the [README](https://github.com/apache/pulsar/tree/master/site2) for making contributions to the website.
 
+#### Updating user logo
+The Apache Pulsar Community welcomes all users to update their company logos on Pulsar website. If your team or organization has adopted Pulsar in production, please update your company logos on [Pulsar website](https://pulsar.apache.org/en/powered-by/).
+
+It only takes 4 steps to update your company logo:
+1. Open [link](https://github.com/apache/pulsar/blob/master/site2/website/data/users.js) in your browser, click “edit” to fork Pulsar repo, create a new branch and edit this file.
```
Comment (Huanli-Meng) [inline site2/website/contributing.md:302]:
```suggestion
1. Open this [link](https://github.com/apache/pulsar/blob/master/site2/website/data/users.js) in your browser, and then click the **Edit** icon (it looks like a pencil) to edit this file.
```

**Review by Huanli-Meng — COMMENTED** [review]
leave my comments, PTAL.

In addition, please pay attention to the indent of contents under each step.

File: site2/website/contributing.md [hunk 9]
```diff
@@ -295,6 +295,30 @@ the static content is generated by CI job and merged into the `asf-site` branch.
...
+ 	       logo: 'https://streamnative.io/static/images/streamnative-logo-white-bakground.png',
+	       logo_white: true
+	    }
+	`
+```	
+>Note
+></br>The `logo_white` parameter is set to true only when your company logo is in white color. Otherwise, you only need to add `name`, `url` and `logo`.
+
+3. Commit the changes in your account’s repo, publish the branch and create pull request.
+4. Update comments, if any.  If no more comment, your pull request will be approved and merged by reviewers.
```
Comment (Huanli-Meng) [inline site2/website/contributing.md:320]:
```suggestion
4. Update comments, if any. If no more comment, your pull request will be approved and merged by reviewers.
```

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Review by sijie — APPROVED** [review]

**Comment by tuhaihe** [comment]
@JipeiWang Thanks for your contribution, that's great.  It will be helpful for operating.

**Pull request merged**